pkg/rancher-desktop/backend/lima.ts

```diff
@@ -671,7 +671,14 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
   }
 
   protected static get limactl() {
-    return path.join(paths.resources, os.platform(), 'lima', 'bin', 'limactl');
+    let limactlBin = 'limactl'
+    // Darwin versions for Ventura start at 22.1.0
+    // https://en.wikipedia.org/wiki/MacOS_Ventura#Release_history
+    if (os.platform() == 'darwin' && semver.gte(os.release(), '22.1.0')) {
+      limactlBin = 'limactl-ventura'
```

Comment:
I think this should be:

```suggestion
      limactlBin = 'limactl.ventura'
```

Reply:
Done.